code/game/turfs/change_turf.dm

```diff
@@ -58,6 +58,13 @@ GLOBAL_LIST_INIT(blacklisted_automated_baseturfs, typecacheof(list(
 /turf/proc/ChangeTurf(path, list/new_baseturfs, flags)
 	if(!path)
 		return
+	if(path == /turf/baseturf_bottom)
```

Comment:
Change this to a switch

Reply:
I want to catch people who try to set baseturf to /turf/open/space/basic

Reply:
I'm not sure what you mean by this, we already have a /basic check later on and a switch wouldn't change any of the existing functionality

Reply:
I mean if someone writes `"Baseturf": "/turf/open/space/basic"` in the JSON that should be corrected

Reply:
Switching this and line 69 to be part of a switch wont change any functionality